### PR TITLE
Fixed #28195 -- Make OSMWidget zoom level configurable

### PR DESCRIPTION
**Pull request by dbrgn**
https://code.djangoproject.com/ticket/28195

### CODE REVIEW

File: tests/gis_tests/test_geoforms.py [hunk 1]
```diff
@@ -346,6 +349,10 @@ class PointForm(forms.Form):
             self.assertNotIn(
                 "options['default_lat'] = %d;" % forms.OSMWidget.default_lat,
                 rendered)
+        if forms.OSMWidget.default_zoom != 17:
```
Comment (dbrgn) [inline tests/gis_tests/test_geoforms.py]: To be honest I don't quite understand that condition, I simply copied it from the two examples above.

**Comment by dbrgn** [comment]
Jenkins complains:

    /home/jenkins/workspace/docs/docs/releases/2.0.txt:78: WARNING: py:class reference target not found: django.contrib.gis.forms.widgets.OSMWidget

Not sure why that's the case.

File: tests/gis_tests/test_geoforms.py (hunk 1 above)
Comment (timgraham) [inline tests/gis_tests/test_geoforms.py]: Agreed, I'll suggest https://github.com/django/django/pull/8493 and then we can rebase this and use that approach if it's okay.

**Comment by dbrgn** [comment]
Great, I'll await #8493 and will then rebase.

I can add documentation for the attributes in a separate PR, but maybe after this one merged? Then all related new docs are in one commit.

**Comment by timgraham** [comment]
Documenting the existing attributes in a separate PR allows easily backporting it to 1.11. Documenting the new attribute in this PR allows the release note to link to it.

**Pull request merged**